package.json

```diff
@@ -122,5 +122,11 @@
   },
   "release": {
     "preset": "conventionalcommits"
+  },
+  "dependencies": {
+    "@linusborg/vue-simple-portal": "^0.1.5",
+    "@popperjs/core": "^2.10.2",
+    "portal-vue": "^2.1.7",
+    "uuid": "^8.3.2"
```

Comment:
Which of these dependencies are truly needed? Vue-simple-portal is a smaller repro of portal-vue. uuid is for generating unique IDs, but I don't think we need it.

Reply:
We can drop uuid and portal-vue (I thought I already had for the latter). 

We need simple portal since this version doesn't break provide/inject.

Reply:
if we can drop uuid and portal-vue let's please do so